### DISCUSSION TITLE
作業繳交延後可能性

### DISCUSSION THREAD

**Discussion by henrytsui000** [bol-edu/HLS-SOC-Discussions#20] [General]
助教大大您好:

考量到最近online FPGA板沒有十分穩定，以及lab2中投影片的連結和實際內容不同(KV260/pynq)讓我和不少學生都需要時間修改內容和完成作業，請問有機會延後作業繳交期限嗎?

**Comment by ciwade870307** [comment]
同學您好,

此問題屬於課程相關，請發佈於課程系統的討論區，謝謝。(因為老師這學期有在台大與交大開課)

**Reply by henrytsui000** [reply]
交大E3沒有開討論區:D

**Reply by henrytsui000** [reply]
<img width="802" alt="image" src="https://user-images.githubusercontent.com/54672031/192082120-3e4d3b7a-f8f6-416d-b8de-614794f8e9f7.png">

"如果對這次作業有任何疑問，請參考以下網站的README.md在網站上發問，或來信詢問助教，謝謝！"
這次作業不算是這次作業嗎？

**Reply by ciwade870307** [reply]
我是台大的助教，不確定這邊提問同學的身份。若交大沒有課程討論區，建議修課相關問題直接寄信給助教

**Reply by henrytsui000** [reply]
您好，我是交大的學生，不好意思或許交大學生較為愚笨或是我是交大學生中特別愚笨的學生，不太理解您的回應，但有點困惑能否延後繳交期限為何不屬於對這次作業的疑問的範疇？

之所以不選擇直接寄信，是因為我相信不只我有我面臨PDF表達不是很完整或是遇到online FPGA無法連線問題（至少我和另外四位朋友因此被影響），所以選擇在discussions區發問。

**Reply by tofumanjeff** [reply]
同學您好，這個討論區還是以建立一個技術討論的社群為主，包含lab實作上的問題、老師上課內容或是其他HLS技術問題，都可以在這邊討論。
但如果是課程規定、作業繳交規範等問題，還是回歸到各校的課程網站或系統，如果貴校課程網站沒有討論區，可以直接寄信給助教，我相信有需要的話助教就會通知所有同學。
